presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -60,7 +60,7 @@
     private boolean legacyOrderBy;
     private boolean legacyMapSubscript;
     private boolean optimizeMixedDistinctAggregations;
-    private boolean distributedSort = false;
+    private boolean distributedSort = true;
```

Comment:
Just enable it in `DistributedQueryRunner`, as it is done for iterative optimizer.